analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/UtilityAnalyzerBaseTests.cs

```diff
@@ -21,31 +21,49 @@
 using System;
 using System.Collections.Immutable;
 using System.Linq;
-using System.Text;
 using FluentAssertions;
 using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.Diagnostics;
 using Microsoft.CodeAnalysis.Text;
+using Microsoft.CodeAnalysis.VisualBasic;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
-using Moq;
+using SonarAnalyzer.Common;
 using SonarAnalyzer.Helpers;
 using SonarAnalyzer.Rules;
+using SonarAnalyzer.UnitTest.Helpers;
 
 namespace SonarAnalyzer.UnitTest.Rules.Utilities
 {
     [TestClass]
     public class UtilityAnalyzerBaseTests
     {
+        private const string DefaultSonarProjectConfig = @"ResourceTests\SonarProjectConfig\Path_Windows\SonarProjectConfig.xml";
+        private const string DefaultProjectOutFolderPath = @"ResourceTests\ProjectOutFolderPath.txt";
+
         [DataTestMethod]
-        [DataRow(LanguageNames.CSharp, @"path\output-cs")]
-        [DataRow(LanguageNames.VisualBasic, @"path\output-vbnet")]
+        [DataRow(LanguageNames.CSharp, DefaultProjectOutFolderPath, @"path\output-cs")]
+        [DataRow(LanguageNames.VisualBasic, DefaultProjectOutFolderPath, @"path\output-vbnet")]
+        [DataRow(LanguageNames.CSharp, DefaultSonarProjectConfig, @"C:\foo\bar\.sonarqube\out\0\output-cs")]
+        [DataRow(LanguageNames.VisualBasic, DefaultSonarProjectConfig, @"C:\foo\bar\.sonarqube\out\0\output-vbnet")]
+        [TestCategory("Utility")]
+        public void ReadConfig_OutPath(string language, string additionalPath, string expectedOutPath)
+        {
+            // We do not test what is read from the SonarLint file, but we need it
+            var utilityAnalyzer = new TestUtilityAnalyzer(language, @"ResourceTests\SonarLint.xml", additionalPath);
+
+            utilityAnalyzer.TestOutPath.Should().Be(expectedOutPath);
+            utilityAnalyzer.TestIsAnalyzerEnabled.Should().BeTrue();
+        }
+
+        [TestMethod]
         [TestCategory("Utility")]
-        public void ReadParameters_OutputPath(string language, string expectedWorkDirectoryPath)
+        public void ReadConfig_OutPath_FromSonarProjectConfig_HasPriority()
         {
             // We do not test what is read from the SonarLint file, but we need it
-            var utilityAnalyzer = new TestUtilityAnalyzer(language, @"ResourceTests\SonarLint.xml", @"ResourceTests\ProjectOutFolderPath.txt");
+            var utilityAnalyzer = new TestUtilityAnalyzer(LanguageNames.CSharp, @"ResourceTests\SonarLint.xml", DefaultProjectOutFolderPath, DefaultSonarProjectConfig);
```

Comment:
it's a nitpick, but could you also provide the additional files in a different order, too (make this a `DataTestMethod`)? to make sure it doesn't count?